src/ResourceManager/IotHub/Commands.IotHub.Test/ScenarioTests/IotHubTests.cs

```diff
@@ -12,6 +12,7 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Microsoft.Azure.Commands.ScenarioTest;
```

Comment:
If nothing else was changed in this file you shouldn't need this using statement.

Reply:
Removed